meta/documents/user_guide/de/test_mode.md

```diff
@@ -0,0 +1,23 @@
+# Wayfair Plugin: Verwendung des Testmodus
+
+## Einführung
+Das Wayfair Plugin verfügt über einen Testmodus zur Bewertung seiner Funktionen, ohne die Live-Produktionsdaten in Wayfair-Systemen zu beeinflussen. **Die Einstellung "Modus" gilt ebenso wie die API-Anmeldeinformationen separat für jedes Plugin-Set in Plentymarkets.**

```

Comment:
I believe "Modus" should not have been translated and should be `Mode`

Reply:
Mode is translated in the screenshot, so I believe it's fine